### PR TITLE
Updating pandas to 1.3.5

### PR DESCRIPTION
**Pull request by sdelarquier**
[Context thread](https://alanhealth.slack.com/archives/CBD6470LD/p1643305254681849)

### CODE REVIEW

File: docker/config/requirements.txt [hunk 1]
```diff
@@ -72,7 +72,7 @@ natsort==7.1.1
 numpy==1.20.2
 openapi-schema-validator==0.1.5
 openapi-spec-validator==0.3.0
-pandas==1.2.4
```
Comment (ssoulier) [inline docker/config/requirements.txt:75]: We should put this in alan-jobs/dags/requirements.txt

**Review by ssoulier — CHANGES_REQUESTED** [review]
everything should happen in alan-jobs

File: docker/config/requirements.txt (hunk 1 above)
Comment (sdelarquier) [reply docker/config/requirements.txt:75]: this was already here, so should I then remove it from the MWAA repo as we can't have 2 requirements with contradictory versions ?

Comment (sdelarquier) [reply docker/config/requirements.txt:75]: From our sync: modifying in our repo works for the prod environment but fails for local install. I'll find a workarround for the local install.

**Pull request closed without merging**